components/filecoin/rs/Cargo.toml

```diff
@@ -9,16 +9,16 @@ license = "MPL-2.0"
 base64 = "0.13.0"
 bls-signatures = { version = "0.12", default-features = false, features = ["pairing"] }
 cxx = "1"
-fvm_shared = { version = "3.10.0" }
+fvm_shared = { version = "^4.5.1" }
```

Comment:
Why ^ qualifier is used here?

Reply:
We don't. Fixed. I've rearranged the changes so the `gnrt` mechanical changes are in a separate commit from the ones done by me.